recipe/meta.yaml

```diff
@@ -277,7 +277,7 @@ outputs:
         # other requirements
         - python
         - typing_extensions >=4.8.0
-        - sympy >=1.13.1
+        - sympy 1.13.1
```

Comment:
sorry for the rapid fire: 
is there a known bug that makes sympy 1.13.3 incompatible with pytorch 2.4 or 2.5? otherwise i would rather unpin.

This kind of `pinning` is great for tight distributions, but it goes against the sharing and interoperability of cocnda-forge.

Reply:
unpin = add a patch to make the  `pip check` more forgiving.

Reply:
It seems there ware test failures on windows and mac with 1.13.2: https://github.com/pytorch/pytorch/pull/133235

Reply:
Not sure if the issue has been resolved in version 1.13.2. Let's patch to allow version 1.13.2 (or > 1.13.2) and see if the failures disappear (uncertain about how to test it yet though..)

Reply:
@jeongseok-meta it may be appropriate to sometimes add `!=1.13.2` and adding the PR you have as a reference.

@atalman are you able to point us to the exact tests that were failing so we can test with 1.13.3?